vpr/test/test_interchange_netlist.cpp

```diff
@@ -0,0 +1,33 @@
+#include "catch2/catch_test_macros.hpp"
```

Comment:
Yes, this is under the `C++ Unit Test` workflow: https://github.com/verilog-to-routing/vtr-verilog-to-routing/runs/4354489589?check_suite_focus=true

Reply:
I just checked the log file of the workflow. I just see 3 tests. Does this test go under any existing tests? Or it should go independently?

![image](https://user-images.githubusercontent.com/11499826/144104861-8c781547-3963-4feb-ad5f-af4209e5eccd.png)

Reply:
There might be an option to increase verbosity which I may add, but the tests are run under `vpr_tests`

Reply:
No worries. Thanks for clarification. If we have the verbosity option, it will be helpful.

Reply:
So, I have checked a bit the options and unfortunately there is no straightforward way of having a bit more verbose output (e.g. with the single test cases names), we may enable full verbosity (which is way too much) or keep the following, which I think is the best option. 

The current setting will output the full log in case of failure though.